src/frontend/src/app/services/websocket/websocket.service.ts

```diff
@@ -56,6 +56,9 @@ export class WebSocketService {
         this.lobbyService.updateCurrentRound(0);
         this.lobbyService.updateEndData(data.endData);
         return;
+      case GameEvent.DISCONNECT:
+        this.router.navigate(['/']);
```

Comment:
Maybe we should also show an indicator that the client joined a different lobby? We could use the material toast or something similar.

Reply:
https://user-images.githubusercontent.com/57273388/217133402-6fffea07-4d80-4a01-aac1-f5bf2752601b.mp4


added toast